model/src/main/resources/db-update/20060-from-2.5.0-to-2.6.0.sql

```diff
@@ -58,4 +58,64 @@ CREATE TABLE build_configuration_align_strats_aud
 
     ALTER TABLE build_configuration_align_strats_aud
         ADD CONSTRAINT fk_buildconfiguration_align_strats_aud_buildconfiguration FOREIGN KEY (buildconfiguration_id, rev) REFERENCES buildconfiguration_aud(id, rev);
+
+--------------------------------------------------------------------------------
+-- DeliverableAnalyzerReport
+--------------------------------------------------------------------------------
+CREATE TABLE delan_report
+    (
+        operation_id        INTEGER                 NOT NULL,
+        PRIMARY KEY (operation_id)
+    );
+    ALTER TABLE delan_report
+        ADD CONSTRAINT fk_delanreport_operation
+            FOREIGN KEY (operation_id)
+            REFERENCES operation(id);
+
```

Comment:
where is the table for Set of labels?

Reply:
As we both already know, Set of labels table not needed, since the converter is used. Hence, resolving this.